package.json

```diff
@@ -1,109 +1,98 @@
 {
   "name": "storybook-addon-playground",
-  "version": "0.0.0",
+  "version": "0.0.1",
   "description": "A playground to enable consumers learn how to use the component library or to reproduce bugs",
+  "type": "module",
   "keywords": [
-    "playground", "sandbox", "code", "debug", "learn", "reproduce", "bugs", "component", "storybook-addons"
+    "playground",
+    "sandbox",
+    "code",
+    "debug",
+    "learn",
+    "reproduce",
+    "bugs",
+    "component",
+    "storybook-addons"
   ],
   "repository": {
     "type": "git",
     "url": "https://github.com/YossiSaadi/storybook-addon-playground"
   },
+  "storybook": {
+    "displayName": "Playground",
+    "supportedFrameworks": [
+      "react"
+    ],
+    "icon": "https://user-images.githubusercontent.com/321738/63501763-88dbf600-c4cc-11e9-96cd-94adadc2fd72.png"
+  },
   "author": "Yossi Saadi <yossisaadi@gmail.com>",
   "license": "MIT",
   "exports": {
-    ".": {
-      "require": "./dist/index.js",
-      "import": "./dist/index.mjs",
-      "types": "./dist/index.d.ts"
-    },
     "./manager": {
-      "require": "./dist/manager.js",
-      "import": "./dist/manager.mjs",
+      "import": "./dist/manager.js",
       "types": "./dist/manager.d.ts"
     },
-    "./preview": {
-      "require": "./dist/preview.js",
-      "import": "./dist/preview.mjs",
-      "types": "./dist/preview.d.ts"
+    "./index": {
+      "import": "./dist/index.js",
+      "types": "./dist/index.d.ts"
     },
     "./package.json": "./package.json"
   },
-  "main": "dist/index.js",
-  "module": "dist/index.mjs",
-  "types": "dist/index.d.ts",
   "files": [
     "dist/**/*",
     "README.md",
     "*.js",
     "*.d.ts"
   ],
   "scripts": {
-    "clean": "rimraf ./dist",
-    "prebuild": "yarn clean",
-    "build": "tsup",
-    "build:watch": "yarn build --watch",
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "run-p build:watch 'storybook --quiet'",
-    "prerelease": "zx scripts/prepublish-checks.mjs",
-    "release": "yarn build && auto shipit",
-    "eject-ts": "zx scripts/eject-typescript.mjs",
+    "build": "rollup -c",
+    "build:watch": "npm run build -- -w",
+    "lint": "eslint .",
+    "lint:fix": "eslint . --fix",
     "storybook": "storybook dev -p 6006",
-    "build-storybook": "storybook build"
+    "start": "npm run build && npm run storybook"
```

Comment:
Shouldn't it be `build:watch`?

Reply:
build:watch causing the process to be stuck, and since vite is fast, if we're running them concurrently, it runs the storybook before dist is published

Reply:
So everytime you make changes while working locally you need to re-build?

Reply:
you can run build:watch regardless
but I'm not sure it would affect the content rendered to Vite